specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/config/server.binding.with.entry.yaml

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: test
+vaults:
+    test0:
+        type: test
+guards:
+    test0:
+        type: test
+        options:
+            credentials: TOKEN
+            lifetime: PT30S
+            challenge: PT5S
+            roles:
+                - read
+                - write
+telemetry:
+    attributes:
+        service.namespace: example
+    metrics:
+        - test.counter
+        - test.gauge
+        - test.histogram
+    exporters:
+        test0:
+            type: test
+bindings:
+    net0:
+        type: test
+        kind: server
+        entry: entry0
```

Comment:
I don't see any reference to `entry0` is this expected?

Reply:
this script is added to test negative scenario(if kind: server, entry property is not allowed). expected output is `JsonException`. So I thought having an entry defined might not be required.